HLTrigger/special/src/SealModule.cc

```diff
@@ -65,5 +66,6 @@ DEFINE_FWK_MODULE(HLTEcalResonanceFilter);
 DEFINE_FWK_MODULE(HLTRegionalEcalResonanceFilter);
 
 DEFINE_FWK_MODULE(HLTPixelAsymmetryFilter);
+DEFINE_FWK_MODULE(HLTPixelActivityHFSumEnergyFilter);
```

Comment:
you are already doing this at the end of the HLTPixelActivityHFSumEnergyFilter.cc file, you should not make any changes in this file.